# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/ladybird/hotfix-skia.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- ladybird/vcpkg.json.vanilla	2024-10-14 03:40:28.914958593 +0100
+++ ladybird/vcpkg.json	2024-10-14 03:40:36.954957304 +0100
@@ -106,7 +106,7 @@
     },
     {
       "name": "skia",
-      "version": "129#0"
+      "version": "133#0"
     },
     {
       "name": "sqlite3",
